src/components/MessageDataContent.tsx

```diff
@@ -7,11 +7,33 @@ import MessageBubbleIcon from '../icons/icon-message-bubble.svg';
 import { FunctionCallData } from '../types';
 import { noop } from '../utils';
 
+const ICON_WIDTH = 16;
+const RIGHT_CONTENT_WIDTH = 92;
+const DATA_ROW_GAP = 8;
+
 const Text = styled.div`
   font-size: 14px;
   font-weight: 500;
   line-height: 20px;
   letter-spacing: -0.1px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  word-wrap: break-word;
+  max-width: calc(
+    100% - ${ICON_WIDTH}px - ${DATA_ROW_GAP}px - ${DATA_ROW_GAP}px -
+      ${RIGHT_CONTENT_WIDTH}px
+  );
```

Comment:
Do we need a max-width?

Reply:
In my opinion, since the width of the parent element is already limited, wouldn't it be preferable for the child elements to be naturally determined?

Reply:
no according to the design this middle component width should be dynamic to its content

Reply:
<img width="458" alt="Screenshot 2024-05-10 at 11 15 17 AM" src="https://github.com/sendbird/chat-ai-widget/assets/16806397/13207a26-d71e-45f2-8f3d-e654b413f705">

Reply:
네, 자식요소의 사이즈는 자유롭게 늘어나야 하는게 맞구요
제가 말씀드린건, 최대 너비는 이미 부모의 사이즈에서 결정되어있기 때문에 저희가 굳이 max-width 를 지정하지 않아도 되지 않나
요 이야기였습니다!

Reply:
오 @bang9 말씀이 맞네요. 없어도 잘 됩니다!!
![Screenshot 2024-05-10 at 1 29 22 PM](https://github.com/sendbird/chat-ai-widget/assets/16806397/80debb51-bc13-43c2-83f1-3a6fbaa0523a)